internal/cli/version.go

```diff
@@ -25,9 +25,8 @@ var (
 const installCLISnippetLinux = `curl -Ls https://download.newrelic.com/install/newrelic-cli/scripts/install.sh | bash && sudo newrelic install`
 const installCLISnippetWindows = `[Net.ServicePointManager]::SecurityProtocol = 'tls12, tls'; (New-Object System.Net.WebClient).DownloadFile(\"https://download.newrelic.com/install/newrelic-cli/scripts/install.ps1\", \"$env:TEMP\\install.ps1\");", "& $env:TEMP\\install.ps1;", "& 'C:\\Program Files\\New Relic\\New Relic CLI\\newrelic.exe' install`
 
-// NewRelicCLILatestReleaseURL is the URL used to fetch the latest release data utilizing GitHub's API.
-// GitHub API Docs: https://docs.github.com/en/rest/reference/repos#get-the-latest-release
-const NewRelicCLILatestReleaseURL string = "https://download.newrelic.com/install/newrelic-cli/currentVersion.txt"
+// NewRelicCLILatestReleaseURL is the URL used to fetch the latest CLI release.
+const NewRelicCLILatestReleaseURL string = "https://s3.us-east-1.amazonaws.com/nr-downloads-main/install/newrelic-cli/currentVersion.txt"
```

Comment:
@ctrombley I think we should not use S3 here, because the most up to date version will be returned. During a fresh cli release, the user would have downloaded a potentially out-dated cli (up to 10/15 minutes) and this would be a consistent endpoint until the CDN cache refreshes.
It's ok for the cli version used by the user to be behind the "true" latest version, it's at least consistent between the check of the version and what the user has downloaded. Eventually the cache will refresh and the user will use latest.

Reply:
👍